components-core/src/main/java/com/adyen/checkout/components/core/ComponentAvailableCallback.kt

```diff
@@ -7,9 +7,6 @@
  */
 package com.adyen.checkout.components.core
 
-import androidx.annotation.RestrictTo
-
-@RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)
 interface ComponentAvailableCallback {
```

Comment:
If this is no longer internal we should probably also make `WeChatPayProvider` public and `PaymentMethodAvailabilityCheck` as well